### PR TITLE
data source iam role

### PR DESCRIPTION
**Pull request by zengchen1024**
=== RUN   TestAccIAMRoleV3DataSource_basic
--- PASS: TestAccIAMRoleV3DataSource_basic (17.38s)
PASS
ok      github.com/terraform-providers/terraform-provider-huaweicloud/huaweicloud       17.388s

### CODE REVIEW

**Comment by zengchen1024** [comment]
@niuzhenguo Could you take a look at this pr. thanks!

**Comment by niuzhenguo** [comment]
@zengchen1024 Please also help to update the CHANGELOG to include the new resources as 551eba2

**Comment by zengchen1024** [comment]
@niuzhenguo  Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ FEATURES:
 * **New Data Source:** `huaweicloud_kms_key_v1`
 * **New Data Source:** `huaweicloud_kms_data_key_v1`
 * **New Data Source:** `huaweicloud_rds_flavors_v1`
+* **New Data Source:** `huaweicloud_iam_role_v3`
```
Comment (niuzhenguo) [inline CHANGELOG.md]: v1.0.0 is already released! this should be added under v1.1.0 above.

Comment (zengchen1024) [reply CHANGELOG.md]: Done

**Pull request merged**